java/vector/pom.xml

```diff
@@ -133,6 +134,37 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+       <artifactId>maven-shade-plugin</artifactId>
+       <version>3.2.1</version>
+       <executions>
+         <execution>
+           <phase>package</phase>
+           <goals>
+             <goal>shade</goal>
+           </goals>
+           <configuration>
+             <artifactSet>
+               <includes>
+                 <include>org.apache.arrow:arrow-format</include>
+                 <include>com.google.flatbuffers:*</include>
+               </includes>
+             </artifactSet>
+             <shadedArtifactAttached>true</shadedArtifactAttached>
+             <shadedClassifierName>shade-format-flatbuffers</shadedClassifierName>
+             <keepDependenciesWithProvidedScope>true</keepDependenciesWithProvidedScope>
+             <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
+             <relocations>
+               <relocation>
+                 <pattern>com.google.flatbuffers</pattern>
+                 <shadedPattern>org.apache.arrow.shaded.com.google.flatbuffers</shadedPattern>
```

Comment:
following the shadedPattern in flight, it would be `arrow.vector.com.google.flatbuffers`, could we stick to this format?

Reply:
done.